Service/src/routes/charts.py

```diff
@@ -8,18 +8,21 @@
 from request_validator import validator
 from request_validator.schemas import charts_schema
 from util.error_map import errors
+from util.db_connection import getConfigFilePath
 
 DAY = '1d'
 MONTH = '1m'
 YEAR = '1y'
 
-IEX_DATETIME_FORMAT = '%Y%m%d %H:%M'
+IEX_DATETIME_FORMAT = '%Y-%m-%d %H:%M'
 IEX_DATE_FORMAT = '%Y-%m-%d'
 DATETIME_FORMAT = '%Y-%m-%d %H:%M:%S'
 
 charts_api = Blueprint('charts_api', __name__)
 
-URL_PREFIX = 'https://api.iextrading.com/1.0/stock/'
+URL_PREFIX = 'https://cloud.iexapis.com/v1/stock/'
+
+CONFIG_FILE_PATH = "/Users/rohithdara/dev/StockDog/Service/src/routes/config.json"
```

Comment:
oh hell no lol, you can't reference your own "rohithdara" folder in this code

you need to have it generic such that I (or anyone else) can pull it down and have it work

This is a blocker for merging

Reply:
Fixed lol